File: Codechef/Practice/Age Limit.cpp
https://www.codechef.com/practice/course/python/PPY10/problems/PYDEBUG01




# cook your dish 
for _ in range(int(input())):
    a=list(map(int,input().split()))
    if a[2] >= a[0] and a[2] < a[1]:
        print('YES')
    else:
        print('NO')

File: Codechef/Practice/Age Limit.go
https://www.codechef.com/practice/course/go/PGOL06/problems/AGELIMIT?tab=statement




package main

import "fmt"

func main() {
    var t int
    fmt.Scanf("%d", &t)

    // Loop for each test case
    for t > 0 {
        var x, y, a int

        // Input for each test case using fmt.Scanf
        fmt.Scanf("%d %d %d", &x, &y, &a)

        // Your code for each test case goes here
        if a >= x && a < y {
            fmt.Println("YES");
        } else{
            fmt.Println("NO");
        }
        t--
    }
}